apps/keyboard/js/layouts/emoji.js

```diff
@@ -0,0 +1,225 @@
+Keyboards.emoji = {
+  label: 'Emoji',
+  types: ['text', 'url', 'email'],
+  menuLabel: 'Emoji',
+  pages: [{   // Page 0 - People
+    emojiRange: [128512, 128591],
```

Comment:
Lucky we never convert layout files to JSON, so instead of creating `emojiRange` here you could actually generate keys in a self-executing function here.